capsules/src/process_console.rs

```diff
@@ -13,6 +13,18 @@
 //!  - 'start n' starts the stopped process with name n
 //!  - 'fault n' forces the process with name n into a fault state
 //!
+//! ### `list` Command Fields:
+//!
```

Comment:
Should this include `PID`, `name`, and `Syscalls` to match the actual output?

Reply:
Not sure how I missed those. Added.